src/worker/vm/imports.ts

```diff
@@ -3,14 +3,90 @@ import * as contrib from '@posthog/plugin-contrib'
 import * as scaffold from '@posthog/plugin-scaffold'
 import * as AWS from 'aws-sdk'
 import crypto from 'crypto'
+import dns from 'dns'
 import * as genericPool from 'generic-pool'
-import fetch from 'node-fetch'
-import * as pg from 'pg'
+import ipRangeCheck from 'ip-range-check'
+import net from 'net'
+import nodeFetch, { RequestInit } from 'node-fetch'
+import pg from 'pg'
 import snowflake from 'snowflake-sdk'
+import url from 'url'
 import * as zlib from 'zlib'
 
+import { IllegalOperationError } from './../plugins/run'
 import { writeToFile } from './extensions/test-utils'
 
+interface URLLike {
```

Comment:
This `vm` file is definitely not the right place for all this IP and `fetch` code (the next ~80 lines).